lib/ansible/modules/cloud/amazon/execute_lambda.py

```diff
@@ -262,7 +262,7 @@ def main():
 
     if invoke_params['InvocationType'] == 'RequestResponse':
         try:
-            results['output'] = json.loads(response['Payload'].read())
+            results['output'] = json.loads(response['Payload'].read().decode('utf8'))
```

Comment:
I think you want to move `to_bytes` inside of the `json.loads`, based on the previous use of placing `.decode` on the `.read`